evap/staff/templates/staff_semester_view.html

```diff
@@ -259,7 +259,7 @@ <h3>
                         </div>
                     </div>
                 </div>
-                <form id="evaluation_operation_form" method="GET" action="{% url 'staff:evaluation_operation' semester.pk %}">
+                <form id="evaluation_operation_form" class="table-responsive" method="GET" action="{% url 'staff:evaluation_operation' semester.pk %}">
```

Comment:
Not sure we want the `table-*` class on the form here, shouldn't we instead apply it to the nested tables?

Reply:
> Make any table responsive across all viewports by wrapping a .table with .table-responsive.

https://getbootstrap.com/docs/5.3/content/tables/#responsive-tables

So the `table-responsive` is actually applied to the (or any) parent element of the table. :)